framework/test/unit/modules/bft/bft_params.spec.ts

```diff
@@ -65,7 +66,9 @@ describe('BFT parameters', () => {
 			};
 			await paramsStore.set(height2Bytes, codec.encode(bftParametersSchema, bftParams2));
 			const batch = db.batch();
-			rootStore.finalize(batch);
+			const smtStore = new SMTStore(db);
+			const smt = new SparseMerkleTree({ db: smtStore, keyLength: 11 });
```

Comment:
`keyLength` is optional, so maybe we dont need it here to specify

Reply:
In the tests, we are using different keyLengths and it throws error so I had to configure it

Reply:
make sense!